src/App.jsx

```diff
@@ -24,17 +25,23 @@ function App() {
           (previousValue) => [...previousValue, ...data.drinks],
         ));
     });
-  }, []);
+  }, setCocktailList.sort);
```

Comment:
2nd argument of useEffect should be an array. I guess you want to keep `[]` instead of `setCocktailList.sort`